src/dataflow/metrics/mod.rs

```diff
@@ -27,10 +27,16 @@ lazy_static! {
     )
     .unwrap();
     pub static ref EVENTS_COUNTER: EventsRead = EventsRead::from(&EVENTS_COUNTER_INTERNAL);
-    pub static ref SCHEDULING_ELAPSED_NS: IntCounterVec = register_int_counter_vec!(
+    pub static ref SCHEDULING_ELAPSED_NS: IntGaugeVec = register_int_gauge_vec!(
         "mz_dataflow_operator_scheduling_elapsed_ns",
         "Nanoseconds spent in each operator",
         &["worker_id", "operator_id"]
     )
     .unwrap();
+    pub static ref SCHEDULING_HISTOGRAM: IntGaugeVec = register_int_gauge_vec!(
+        "mz_dataflow_operator_scheduling_histogram",
+        "How many times it took each operator to complete in at most le_ns seconds",
```

Comment:
I don't really understand this sentence. Do you mean something like: `How many times each operator completed in at most le_ns seconds`?

Reply:
ah yeah that is what I meant